docs/unit.md

```diff
@@ -2,32 +2,8 @@
 
 An overview of the tools used by this boilerplate for unit testing:
 
-- [Karma](https://karma-runner.github.io/): the test runner that launches browsers, runs the tests and reports the results to us.
```

Comment:
@eddyerburgh Since we are leaving the karma setup available, I don't think we should remove this docs section. 

We should rather leave that docs in, but prepend the page with new infos about jest.